tests/rptest/tests/admin_uuid_operations_test.py

```diff
@@ -82,18 +89,211 @@ def _uuids_updated():
                 old_uuid = n['uuid']
 
         # get current node id and UUID
-        current = admin.get_broker_uuid(to_stop)
+        current = self.admin.get_broker_uuid(to_stop)
 
-        admin.override_node_id(to_stop,
-                               current_uuid=current['node_uuid'],
-                               new_node_id=initial_to_stop_id,
-                               new_node_uuid=old_uuid)
+        self.admin.override_node_id(to_stop,
+                                    current_uuid=current['node_uuid'],
+                                    new_node_id=initial_to_stop_id,
+                                    new_node_uuid=old_uuid)
 
         self.redpanda.restart_nodes(to_stop,
                                     auto_assign_node_id=True,
                                     omit_seeds_on_idx_one=False)
 
-        after_restart = admin.get_broker_uuid(to_stop)
+        after_restart = self.admin.get_broker_uuid(to_stop)
 
         assert after_restart['node_id'] == initial_to_stop_id
         assert after_restart['node_uuid'] == old_uuid
+
+    def scrape_uuid(self, node: ClusterNode) -> str | None:
+        UUID_LOG = "Generated new UUID for node"
+        lines = [
+            s.strip() for s in self.log_searcher._capture_log(node, UUID_LOG)
+            if UUID_LOG in s
```

Comment:
nit: isn't `UUID_LOG in s` provided by `_capture_log`?

Reply:
not as written. the input string to `_capture_log` is a grep expression; I'd dropped the quotes, lazily fixed it in the list comprehension, and forgot to circle back.

Reply:
fixed